code/game/objects/items/blueprints.dm

```diff
@@ -19,22 +19,22 @@
 	var/const/ROOM_ERR_SPACE = -1
 	var/const/ROOM_ERR_TOOLARGE = -2
 
-/obj/item/blueprints/attack_self(mob/user as mob)
-	if (use_check_and_message(user, USE_DISALLOW_SILICONS))
+/obj/item/blueprints/attack_self(mob/user)
+	if(use_check_and_message(user, USE_DISALLOW_SILICONS))
 		return
 	add_fingerprint(user)
 	interact()
 	return
 
 /obj/item/blueprints/Topic(href, href_list)
 	..()
-	if ((usr.restrained() || usr.stat || usr.get_active_hand() != src))
+	if((usr.restrained() || usr.stat || usr.get_active_hand() != src))
```

Comment:
Is there a reason you arnt using `use_check_and_message` or` use_check `here?

Reply:
updated